encoding/x/protobuf/types.go

```diff
@@ -146,6 +148,19 @@ func NewOnewayHandler(
 	return newOnewayHandler(handleOneway, newRequest)
 }
 
+// ClientBuilderOptions returns ClientOptions that yarpc.InjectClients should use for a
+// specific client given information about the field into which the client is being injected.
+func ClientBuilderOptions(_ transport.ClientConfig, structField reflect.StructField) []ClientOption {
+	var opts []ClientOption
+	for _, opt := range uniqueLowercaseStrings(strings.Split(structField.Tag.Get("proto"), ",")) {
+		switch opt {
+		case "use_json":
```

Comment:
Do we have a better name for this? Would `useJSON` or just `json` be more consistent with Go's naming conventions?